dvc/repo/checkout.py

```diff
@@ -34,36 +34,38 @@ def _checkout(
 ):
     from dvc.stage import StageFileDoesNotExistError, StageFileBadNameError
 
-    stages = set()
-
     if not targets:
         targets = [None]
         _cleanup_unused_links(self)
 
+    pairs = []
```

Comment:
We used `set()` before, now it's a list. Did we lost dedup?

Reply:
Yep. Will fix.